Makefile

```diff
@@ -0,0 +1,53 @@
+include .env
+
+RED := \033[31m
+GREEN := \033[32m
+NC := \033[0m
+
+.NOTPARALLEL:
+
+.PHONY: all
+all: test
```

Comment:
It might make sense to make this `all: distclean test` to ensure that a naive checkout and run `make all` works as expected. WDYT?

Reply:
Have added and tested the Prow jobs, see status checks below 🎉